.github/workflows/python-app.yml

```diff
@@ -46,7 +46,7 @@ jobs:
 
     - name: Test with pytest
       run: |
-        python -m coverage run -m pytest  tests/test_server_basic.py -sv --full-trace --log-cli-level=DEBUG -m 'not isgri_plugin and not dda'
+        python -m coverage run --source=cdci_data_analysis -m pytest tests -sv --full-trace --log-cli-level=DEBUG -m 'not isgri_plugin and not dda'
```

Comment:
Does this guarantee that for the coverage evaluation, only the sources within cdc_data_analysis are considered?
If so, then it's really good!

Reply:
Yes, else you can see the [report](https://codecov.io/gh/oda-hub/dispatcher-app/tree/ec557543c3415dbc6ebf79e2cbaf8e44a96f4754)  it includes other things, e.g. tests themselves.